lib/kraken.js

```diff
@@ -0,0 +1,80 @@
+import KrakenClient from 'kraken-api';
+import CloudWatch from './cloudWatch';
+
+export default class Kraken {
+  constructor() {
+    this.kraken = new KrakenClient(
+      process.env.KRAKEN_API_KEY || 'REPLACEME',
+      process.env.KRAKEN_API_SECRET || 'REPLACEME'
+    );
+    this.cloudwatch = new CloudWatch();
+    this.currentAccount = null;
+    this.currentPaymentMethod = null;
+    this.cryptoCurrency = process.env.PREFERRED_CRYPTO_CURRENCY || 'ETH';
+    this.cryptoCurrencies = ['XBT', 'ETH', 'LTC'];
+    this.localCurrency = process.env.PREFERRED_LOCAL_CURRENCY || 'EUR';
+    this.preferredWallet = process.env.PREFERRED_WALLET || 'fiat_account';
+    this.buyAmount = process.env.BUY_AMOUNT || 1;
+    this.sellAmount = process.env.SELL_AMOUNT || 1;
+  }
+
+  getBalance() {
```

Comment:
Think it should be inside https://github.com/nicka/coinboss/pull/3 if we're going to merge it. For now, I think the balance does not add that much value.